packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -1921,7 +1921,7 @@ export class GetAssetHolding extends Op {
         value = BigInt(assetInfo.amount);
         break;
       case "AssetFrozen":
-        value = stringToBytes(assetInfo["is-frozen"]);
+        value = BigInt(assetInfo["is-frozen"]);
```

Comment:
assetInfo["is-frozen"] ?  1n : 0n;
how about using the above ?
i think this is more intuitive.

Reply:
I don't have a particular preference. But will update if you think this is more intuitive

Reply:
I like @amityadav0  suggestion.